tests/E2E/test/selectors/BO/catalogpage/category_submenu.js

```diff
@@ -1,44 +1,44 @@
+    picture: '//*[@id="category_cover_image"]',
+    thumb_picture: '//*[@id="category_thumbnail_image"]',
+    thumb_menu_picture: '//*[@id="category_menu_thumbnail_images"]',
     upload_files_button: '//*[@id="thumbnail-upload-button"]',
     thumbnail_success_alert: '//*[@id="thumbnail-success"]',
     thumbnail_menu_img: '//*[@id="thumbnail-images-thumbnails"]//img',
-    title: '//*[@id="meta_title_1"]',
-    meta_description: '//*[@id="meta_description_1"]',
-    keyswords: '(//*[@id="fieldset_0"]//input[@placeholder="Add tag"])[1]',
-    simplify_URL_input: '//*[@id="link_rewrite_1"]',
-    save_button: '//*[@id="category_form_submit_btn"]',
-    reset_button: '//*[@id="table-category"]//th[7]//button[contains(@name, "Reset")]',
-    search_input: '//*[@id="table-category"]//input[@name="categoryFilter_name"]',
-    search_button: '//*[@id="submitFilterButtoncategory"]',
-    search_result: '//*[@id="table-category"]//td[3]',
-    update_button: '//*[@id="table-category"]//td[7]/div/div',
-    action_button: '//*[@id="table-category"]//td[7]//button[contains(@class, "dropdown-toggle")]',
-    delete_button: '//*[@id="table-category"]//td[7]//a[@title="Delete"]',
-    view_button: '//*[@id="table-category"]//td[7]//a[@title="View"]',
-    image_link: '//*[@id="image-images-thumbnails"]/div/img',
-    thumb_link: '//*[@id="thumb-images-thumbnails"]/div/img',
-    second_delete_button: '//*[@id="content"]//form//button[not(@name)]',
-    select_category: '//*[@id="table-category"]/tbody/tr/td[1]/input',
-    action_group_button: '//*[@id="form-category"]//div[contains(@class, "bulk-actions")]/button[contains(@class, "dropdown-toggle")]',
-    delete_action_group_button: '//*[@id="form-category"]//a[contains(@onclick, "Delete selected")]',
-    category_number_span: '//*[@id="form-category"]//span[@class="badge"]',
-    category_description: '//*[@id="category_form"]//div[@class="mce-tinymce mce-container mce-panel"]',
-    description: '//*[@id="table-category"]//td[4]',
-    delete_tag_button: '//div[@class="tagify-container"]//span[%POS]/a',
-    mode_link_disable_radio: '//*[@id="deleteMode_linkanddisable"]',
-    mode_link_radio: '//*[@id="deleteMode_link"]',
-    mode_delete_radio: '//*[@id="deleteMode_delete"]',
+    title: '//*[@id="category_meta_title_1"]',
+    meta_description: '//*[@id="category_meta_description_1"]',
+    keyswords: '//*[@id="category_meta_keyword_1-tokenfield"]',
+    simplify_URL_input: '//*[@id="category_link_rewrite_1"]',
+    save_button: '//form[@name="category"]//*[contains(@class,"card-footer")]/button',
+    reset_button: '//*[@id="category_grid"]//thead//button[@type="reset"]',
+    search_input: '//input[@id="category_name"]',
+    search_button: '//button[@name="category[actions][search]"]',
+    search_result: '//*[@id="category_grid"]//td[3]',
+    update_button: '//*[@id="category_grid"]//tbody//td[7]//a[@data-original-title="Edit"]',
```

Comment:
will be broken if we add data in fixtures because of `td[7]` 🤔

Reply:
change it
We will work together on a id in the migration project